modules/core/src/main/scala/ch/epfl/bluebrain/nexus/rdf/Node.scala

```diff
@@ -399,10 +399,10 @@ object Node {
     final def apply(value: Double): Literal =
       new Literal(value.toString, xsd.decimal)
 
-    final implicit def literalShow(implicit is: Show[AbsoluteIri]): Show[Literal] = Show.show {
-      case Literal(f, _, Some(LanguageTag(tag))) => s""""$f"@$tag"""
-      case l @ Literal(f, _, _) if l.isString    => s""""$f""""
-      case Literal(f, dt, None)                  => s""""$f"^^<${is.show(dt)}>"""
+    final implicit def literalShow(implicit is: Show[AbsoluteIri], ls: Show[LanguageTag]): Show[Literal] = Show.show {
+      case Literal(f, _, Some(tag))           => s""""$f"@${ls.show(tag)}"""
```

Comment:
As far as I see on the Json playground if there is languange tag and also `dataType`, the language tag gets ignored. That happens when you try to add a languange tag to a a integer type for example

Reply:
As per the spec:
> A literal in an RDF graph consists of two or three elements:
> *   a lexical form, being a Unicode [UNICODE] string, which should be in Normal Form C [NFC],
> *   a datatype IRI, being an IRI identifying a datatype that determines how the lexical form maps to a literal value, and
> *   if and only if the datatype IRI is http://www.w3.org/1999/02/22-rdf-syntax-ns#langString, a non-empty language tag as defined by [BCP47]. The language tag must be well-formed according to section 2.2.9 of [BCP47].

The language tag should only be present for `http://www.w3.org/1999/02/22-rdf-syntax-ns#langString`.